### DISCUSSION TITLE
target #105 - Ryuk

### DISCUSSION THREAD

**Discussion by Albertinou** [cssbattle/forum#242] [Q&A]
target #105 - Ryuk 

Lately im always getting 99.99%. BUT. On this one, im only getting 96% and it doesnt look any worse than those 99ers. Am i allowed to post my code here for you to check it out?

edit
 Btw, first place isnt doing well, with ~519 points, so i guess its something wrong with this one

**Comment by JDMeijers** [comment]
Yea, got a 100% visual match too, but the colors still seem to cause some problems

**Comment by RazvanManolache** [comment]
Same, but i actually like guessing to see what's wrong. Don't fix it. :)

**Comment by kushsolitary** [comment]
We have identified the issue with the target and have pushed a fix. Should be live in a few minutes.

**Reply by RazvanManolache** [reply]
noooooooooooooo :) well Jeroen already beat me score :p
